animations/index.md

```diff
@@ -1,159 +1,88 @@
+
+```Dart
+import 'package:flutter/animation.dart';
+import 'package:flutter/material.dart';
+```
+For information on the SDK libraries and available widgets in each library, see the following resources.
+
+* [Flutter SDK libraries](https://docs.flutter.io/index.html)  
+* [Animation and Motion Widgets](/widgets/animation/)  
+* [Flutter Widget Index](/widgets/widgetindex/)  
+* [Widget Catalog](/widgets/)
+
+## Flutter animation resources  
+To learn about Flutter animations, check out the following resources.
+
+### Concepts  
+For information about how Flutter animation works and definitions of the main animation concepts see [Flutter Animation Concepts](/animations/concepts.html). You'll also find information here on the main  animation widgets such as the `AnimationController` or `AnimationBuilder`, and information about tweens, tickers, and more.  
+
+### Animation and Motion widgets  
+For a list of Flutter SDK animation widgets, see the following:
+* [Implicit animation widgets](/animations/implicit_widgets.html)  
+* [Transition animation widgets](/animations/transitions_widgets.html)  
+* [Explicit animation widgets](/animations/explicit_widgets.html)  
+* [Animation and Motion Widgets Catalog](/widgets/animation/)  
+
+### Codelabs  
+Codelabs are a great way to learn how to use Flutter and how to add Flutter animation widgets to your app. Check out [Building Beautiful UIs with Flutter](/codelabs/). This Codelab demonstrates how to build a simple chat app. And Step 7 (Animate your app) shows how to animate the new message—sliding it from the input area up to the message list. For more information, see [Codelabs](/codelabs/).  
+If you're new to Flutter, check out the [Write your first app, part 1](https://codelabs.developers.google.com/codelabs/first-flutter-app-pt1/#0) and [Write your first app, part 2](https://codelabs.developers.google.com/codelabs/first-flutter-app-pt2/#0) codelabs. They walk you through the basics of using Flutter to quickly create an iOS and Android app.  
+
+### Flutter animation demos and examples  
+Flutter animation demos and examples provide code samples of some of the common and most popular animation widgets.  
+* [AnimatedList](/catalog/samples/animated-list/)—This example demonstrates how to display a list of cards which stay in sync using the  ListModel widget. When an item is added or removed from the model, the corresponding card animates in or out of view.
+* [Hero Animations](/animations/hero-animations/)—This demo shows how to build standard hero animations, and hero animations that transform the image from a circular shape to a square shape during flight. The [Shrine Demo](https://github.com/flutter/flutter/blob/master/examples/flutter_gallery/lib/demo/shrine_demo.dart) shows another example showing hero animations.
```

Comment:
1. [Hero Animations](...) guide. How to build hero animations (also called shared element animations). The guide shows two examples: a standard hero animation, and one that transforms the image ....

Reply:
Please use "Shared element transition" to refer to this animation pattern. "Hero" is the widget used to implement this kind of animation, and it's a term specific to Flutter. In contrast, "Shared Element Transitions" are known to Android developers as well (e.g., https://android-developers.googleblog.com/2018/02/continuous-shared-element-transitions.html).